docs/api/middleware/probechecker.md

```diff
@@ -0,0 +1,98 @@
+---
+id: probecheker
+title: probecheker
+---
+
+Liveness and readiness probes middleware for [Fiber](https://github.com/gofiber/fiber) that provides two endpoints for checking the health and ready state of any HTTP application.
+
+The endpoint values default to `/livez` for liveness and `/readyz` for readiness. Both functions are optional, the liveness endpoint will return `true` right when the server is up and running but the readiness endpoint will not answer any requests if an `IsReady` function isn't provided. 
```

Comment:
Readiness should default to "200 Ok". If the function can execute it probably means the server us up and responding.

Reply:
> Readiness should default to "200 Ok". If the function can execute it probably means the server us up and responding.

Then what will be difference between ready and live?

Reply:
This could produce some false positives, since the is ready function should only return 200 OK if the server is fully ready to answer any requests and there's no way to check by default all the requirements for readiness, since it involves a lot of different checks (like connections to other servers or some sort of cleanup or starting task).

And there's a way of configuring the liveness and readiness independently with most of the containerized environments, so this shouldn't be a problem.

Reply:
@luk3skyw4lker @efectn It would just be for the default. There's a use case where you dont want to check anything, just get a 200 to mark the pod/container as "green". Else we would be forcing users to define their readiness function.